cranelift/codegen/meta/src/shared/entities.rs

```diff
@@ -15,6 +15,14 @@ pub(crate) struct EntityRefs {
     /// This is primarliy used in control flow instructions.
     pub(crate) block: OperandKind,
 
+    /// A reference to a basic block in the same function.
+    /// This is primarliy used in control flow instructions.
```

Comment:
I see this was copy-pasted from the comment above because they all have the same typo. :laughing: "primarliy" -> "primarily"